ballerina/admin.bal

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) 2023 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java as java;
+
+# Ballerina Service Bus connector provides the capability to access Azure Service Bus SDK.
+# Service Bus API provides data access to highly reliable queues and publish/subscribe topics of Azure Service Bus with deep feature capabilities.
+@display {label: "Azure Service Bus Administrator", iconPath: "icon.png"}
+public isolated client class ASBAdministration {
```

Comment:
Shall we update the client class name to be consistent with the other clients we have already?
```suggestion
public isolated client class AdminClient {
```

Reply:
Sure.. Addressed in [58554cf](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/180/commits/58554cf7b0e15eb59ad5eb45be40a2fd3cddd181).